### DISCUSSION TITLE
A place to put Maps of areas around the world

### DISCUSSION THREAD

**Discussion by coolkid0281** [niels747/2D-Weather-Sandbox#106] [Show and tell]
I will start, I made a Map Of Lake Erie going all the way up to Buffalo, NY and the land ends in Rochester, NY.
Here is the link to download: https://www.dropbox.com/scl/fi/h580x5f43o65tzpzjt3m4/Lake-erie.weathersandbox?rlkey=zc2ygvp2xl1di1kvuda0pbudi&dl=0

**Comment by HurricaneHugo** [comment]
Cool. I created 2000+mph winds. 
![Screenshot 2024-08-17 3 43 31 PM](https://github.com/user-attachments/assets/1aafe8b1-cf47-4c84-85b8-a8a58389590f)
